nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/FXMLController.java

```diff
@@ -696,6 +679,28 @@ public void addDataset(DatasetBase dataset, boolean appendFile, boolean reload)
         getActiveChart().layoutPlotChildren();
     }
 
+    /**
+     * Updates the SpectrumStatusBar menu options based on whether FID mood is on, and the dimensions
+     * of the dataset.
+     */
+    private void updateSpectrumStatusBarOptions() {
+        if (isFIDActive()) {
```

Comment:
When in grid display, when selecting a 1D vs 2D vs FID etc graph the tool bar options on the bottom should update.